docs/FAQ/identity_management_FAQ.md

```diff
@@ -1,18 +1,18 @@
 ## Identity Management (Membership Service)
 &nbsp;
 ##### What is unique about the fabric's Membership Service module?
-The design and implementation of the Membership Service module encompass many of the latest advances in cryptography, which we believe make it stand apart from other alternatives.
+One of the things that makes the Membership Service module stand out from the pack is our implementation of the latest advances in cryptography.
```

Comment:
Omit the entire question and the entire answer or just that phrase? 

I think simply by rephrasing the question without a reference to a direct or implicit comparison you have kept what's useful about the answer without materially changing the question all that much. 

I don't know how much the questions have been written in stone, however.